dynamodb/Acornfile

```diff
@@ -0,0 +1,175 @@
+name:        "AWS DynamoDB"
+description: "Amazon's fast and flexible NoSQL database"
+info:        "\(localData.info)"
+readme:      "./README.md"
+icon:        "./ddb.svg"
+
+args: {
+	// Name to assign the table during creation.
+	tableName: ""
```

Comment:
can we default to `acorn`? The goal should be the user can always just do `acorn run [image]` with no args.

Reply:
The arg isn't required. It uses the default CDK value.